recovery/elliptics_recovery/dc_server_send.py

```diff
@@ -0,0 +1,206 @@
+import logging
+import os
+import errno
+import time
+
+from elliptics_recovery.utils.misc import dump_key_data, load_key_data_from_file
+
+import elliptics
+
+log = logging.getLogger()
+
```

Comment:
This is not a documentation, this is a one-liner which says the purpose of the function without description of the logic and idea behind it. Its like saying 'this is a table' pointing your finger to table. You do not say that it is a wood table or that it is needed to host plates or books or why did you bought it and put it exactly here.

In particular, bucket/server_send classes completely lack description of the recovery process when server sending fails. This is just one note among others related to documentation. Please write not only what particular entity does, but what did you want it to do, how you want it to behave and why is this needed.